scripts/pipelines/build.cross-toolchain.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+SCRIPT_ROOT=$(realpath $(dirname $BASH_SOURCE))
+
+. "${SCRIPT_ROOT}/build.common.sh" "$@"
+
+echo "INFO: Setting SDKMACHINE to x86_64-mingw32"
+sed -i '/^SDKMACHINE/s/"x86_64"/"x86_64-mingw32"/' ./conf/local.conf
```

Comment:
Instead of using `sed` - which is relatively unreliable - can we instead add `SDKMACHINE` to `BB_ENV_EXTRAWHITE`/`BB_ENV_PASSTHROUGH_ADDITIONS` and then just set its value in the calling script, in that same way that you would set the `MACHINE`?